adlfs/multithread.py

```diff
@@ -327,7 +327,7 @@ def run(self, nthreads=None, monitor=True):
         for (rfile, lfile), dic in self.progress.items():
             unique = dic['uuid']
             if len(dic['waiting']) > 1:
-                parts = [self.temp_upload_path+unique+"_%i" % i for i
+                parts = [self.temp_upload_path+unique+"/part_%i" % i for i
```

Comment:
Does this create a folder for each part that is uploaded? I would think we want to create one folder for all the parts to go into instead, right?